management/src/main/scala/akka/management/scaladsl/AkkaManagement.scala

```diff
@@ -86,8 +76,11 @@ final class AkkaManagement(implicit private[akka] val system: ExtendedActorSyste
   private def providerSettings: ManagementRouteProviderSettings = {
     // port is on purpose never inferred from protocol, because this HTTP endpoint is not the "main" one for the app
     val protocol = "http" // changed to "https" if ManagementRouteProviderSettings.withHttpsConnectionContext is used
+    val host = settings.Http.Hostname
+    val port = settings.Http.Port
+    val path = settings.Http.BasePath.fold("")("/" + _)
     val selfBaseUri =
-      Uri(s"$protocol://${settings.Http.Hostname}:${settings.Http.Port}${settings.Http.BasePath.fold("")("/" + _)}")
+      Uri.from(scheme = protocol, host = host, port = port, path = path)
```

Comment:
So if we don't parse it from a generated string ipv6 works out of the box? (Great if so)

Reply:
I have tested in a worksheet. Code like:
```scala
import akka.http.scaladsl.model.Uri

Uri.from(host = "2001:0db8:0000:0000:0000:8a2e:0370:7334")
Uri.from(host = "[2001:0db8:0000:0000:0000:8a2e:0370:7334]")
```

Result like:
```
res0: akka.http.scaladsl.model.Uri = //[2001:0db8:0000:0000:0000:8a2e:0370:7334]
res1: akka.http.scaladsl.model.Uri = //[2001:0db8:0000:0000:0000:8a2e:0370:7334]
```

But if using code like `Uri("http://2001:0db8:0000:0000:0000:8a2e:0370:7334")`, it will throw an exception:
```
akka.http.scaladsl.model.IllegalUriException: Illegal URI reference: Unexpected end of input, expected userinfo-char, pct-encoded or '@' (line 1, column 47): http://2001:0db8:0000:0000:0000:8a2e:0370:7334
                                              ^
  at akka.http.scaladsl.model.IllegalUriException$.apply(ErrorInfo.scala:74)
  at akka.http.scaladsl.model.Uri$.fail(Uri.scala:831)
  at akka.http.impl.model.parser.UriParser.fail(UriParser.scala:81)
  at akka.http.impl.model.parser.UriParser.parseUriReference(UriParser.scala:46)
  at akka.http.scaladsl.model.Uri$.apply(Uri.scala:216)
  at akka.http.scaladsl.model.Uri$.apply(Uri.scala:188)
  ... 36 elided
```

Reply:
Good, thanks for confirming!